pkg/subcmd/template.go

```diff
@@ -55,10 +55,10 @@ Examples:
   $ tssc template --show-manifests=false
 
   # Rendering only the templates of a single Helm Chart.
-  $ tssc template --show-values=false charts/rhtap-subscriptions
+  $ tssc template --show-values=false tssc/rhtap-subscriptions
```

Comment:
this should not rename charts it should rename rhtap

Reply:
Done